addons/website_blog/views/website_blog_templates.xml

```diff
@@ -487,5 +495,55 @@
          <li><a href="#" data-action="new_blog_post">New Blog Post</a></li>
     </xpath>
 </template>
+
+<!-- RSS Feed -->
+<template id="website_blog_rss" name="Blog RSS">&lt;?xml version="1.0" encoding="utf-8"?&gt;
+    <rss xmlns:content="http://purl.org/rss/1.0/modules/content/" xmlns:atom="http://www.w3.org/2005/Atom" version="2.0">
+        <channel>
+            <title><t t-esc="blog.name" /></title>
+            <link t-att-href="'%s/blog/%s' % (base_url, slug(blog))" rel="alternate" />
+            <description><t t-esc="blog.name" /></description>
+            <atom t-att-href="'%s/blog/%s/atom' % (base_url, slug(blog))" rel="self" type="application/atom+xml" />
+            <lastBuildDate><t t-esc="blog.write_date" /></lastBuildDate>
+            <item t-foreach="blog.post_ids[:limit]" t-as="post">
+                <title><t t-esc="post.name" /></title>
+                <link t-att-href="base_url + post.website_url" rel="alternate" />
+                <description><t t-esc="post.subtitle" /></description>
+                <content><t t-esc="post.content" /></content> 
+                <author>
+                    <name><t t-esc="post.author_id.sudo().name" /></name>
```

Comment:
@rim-odoo because the public user can not read by Rss Feed author (name and email) then I used <code>sudo()</code>